crates/node/src/interactive_cli/context/identity.rs

```diff
@@ -47,6 +48,28 @@ enum ContextIdentitySubcommands {
         )]
         force: bool,
     },
+    #[command(about = "Grant permissions to a member")]
+    Grant {
+        #[arg(long, short, default_value = "default")]
+        context: Alias<ContextId>,
+        #[arg(long = "as", default_value = "default")]
+        granter: Alias<PublicKey>,
+        #[arg(help = "The member to grant permissions to")]
+        grantee: PublicKey,
+        #[arg(help = "The capability to grant")]
+        capability: Capability,
+    },
+    #[command(about = "Revoke permissions from a member")]
+    Revoke {
+        #[arg(long, short, default_value = "default")]
+        context: Alias<ContextId>,
+        #[arg(long = "as", default_value = "default")]
+        revoker: Alias<PublicKey>,
+        #[arg(help = "The member to revoke permissions from")]
+        revokee: PublicKey,
+        #[arg(help = "The capability to revoke")]
+        capability: Capability,
+    },
```

Comment:
this will be `context identity grant` btw, which isn't bad, but different from what you have with `meroctl` as `context grant`, I think I like `context identity grant`

Reply:
@miraclx Done.

Reply:
Oh, I see now, thanks for the explanation!